sql/moz-fx-data-shared-prod/apple_ads/ad_group_report/view.sql

```diff
@@ -1,9 +1,4 @@
--- This is an authorized view that allows us to read Apple Ads data
--- from a Fivetran project.
-CREATE OR REPLACE VIEW
-  `moz-fx-data-shared-prod.apple_ads.ad_group_report`
-AS
```

Comment:
The `view.sql` files still need to have `CREATE OR REPLACE VIEW`.

Reply:
🤦 yes, I'm not sure why I deleted this 😬